src/reducers/index.js

```diff
@@ -1266,4 +1268,25 @@ export const onNanoContractEditAddress = (state, { payload }) => {
   };
 };
 
+/**
+ * @param {Object} state
+ * @param {{
+ *   payload: string (ncId)
+  * }} action
+ */
+export const onNanoContractUnregister = (state, { payload }) => {
+  if (!(payload in state.nanoContracts)) {
+    // This should never happen, we are trying to unregister
+    // a nano contract that is not registered
+    return state;
+  }
+
+  const {[payload]: _, ...newNanoContracts} = state.nanoContracts;
```

Comment:
I know I made the suggestion of this syntax but I think we should document this more.

```suggestion
/**
 * Remove the given nano contract from the registered nano contracts state.
 *
 * @param {Object} state
 * @param {Object} action
 * @param {string} action.payload
 * @returns {Object}
 */
export const onNanoContractUnregister = (state, { payload }) => {
  if (!(payload in state.nanoContracts)) {
    // This should never happen, we are trying to unregister
    // a nano contract that is not registered
    return state;
  }

  // Create a new object with all keys from state.nanoContracts except `payload`
  const {[payload]: _, ...newNanoContracts} = state.nanoContracts;
```

Reply:
Done https://github.com/HathorNetwork/hathor-wallet/pull/625/commits/4b3f802c4b1a5ad9de5362b499762673924bb097